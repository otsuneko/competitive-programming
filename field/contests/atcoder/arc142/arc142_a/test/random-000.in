705055195 150152796
