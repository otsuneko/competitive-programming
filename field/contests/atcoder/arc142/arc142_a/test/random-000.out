150152796 697251051 150152796
697251051 150152796 697251051
2
